### DISCUSSION TITLE
Automazione presenza in casa con ssid

### DISCUSSION THREAD

**Discussion by djfoxmiki** [indomus/forum#2021] [GENERALE - Discussioni generiche]
Ciao a tutti ragazzi, ho necessità di configurare una volta per tutte HA per automazioni che dovrebbero eseguirsi in occasione o meno della presenza/assenza CERTA degli inquilini della mia abitazione.
Attualmente sto utilizzando il gps e l’app con due dispositivi iPhone. Tuttavia, il tempismo e l’imprecisione del segnale mi impedisce di avere automazioni corrette.
Pertanto vorrei capire se è possibile configurare il tutto attraverso l’entità SSID, quindi quando i dispositivi siano o meno connessi alla mia rete domestica.

Ringrazio tutti per l’attenzione e l’eventuali risposte.

Saluti

**Comment by blakdog1978** [comment]
Certo che si può io uso nmap tracker

Il ven 28 ott 2022, 20:53 djfoxmiki ***@***.***> ha scritto:

**Reply by djfoxmiki** [reply]
Grazie mille. 

Mi spieghi cosa devo fare.

**Comment by blakdog1978** [comment]
Comincia ad installarlo e configurarlo segui qualche guida se non l'hai già
fatto

Il ven 28 ott 2022, 22:19 djfoxmiki ***@***.***> ha scritto:

**Reply by djfoxmiki** [reply]
L’ho installato. Provato a seguire qualche guida ma nulla :(

**Comment by blakdog1978** [comment]
Nmap ti trova delle entità? Giusto?...
Devi tra le entità cercare i tuoi dispositivi, li trovi l'indirizzo delle
schede di rete non gli IP.
Poi quando li hai trovati per comodità li rinomini e li abiliti in modo da
avere dei traker funzionanti...

Il ven 28 ott 2022, 23:07 djfoxmiki ***@***.***> ha scritto:

**Reply by djfoxmiki** [reply]
Mi dice che sono entità non disponibili. 

Che procedura devo fareeee. Non trovo una guida. Riesci a spiegarmi per bene?


![DA70C590-643D-4809-9DC7-E3B23F9A8B0A](https://user-images.githubusercontent.com/81266942/198850904-5589a574-0b09-4285-ac27-fa442f32f96f.png)
![3FB46250-E5B2-4B38-A613-2814305394C5](https://user-images.githubusercontent.com/81266942/198850909-b3d40fff-4d14-4cdf-9a59-22f135ce3716.png)

**Reply by blakdog1978** [reply]
Clicca sopra a una -impostazioni -avanzate -abilita
![Screenshot_2022-10-30-07-31-49-461-edit_io homeassistant companion android](https://user-images.githubusercontent.com/36457031/198865790-2824dd79-4f34-425e-a8f4-a1883dfce563.jpg)

**Reply by djfoxmiki** [reply]
Perfetto. Grazie mille. Ho abilitato l’entità. Ma come faccio ad indicare all’automazione che nel momento in cui si connette ad una specifica rete (ne ho diverse) allora significa che sono in casa e fa scattare l’automazione? Cioè non riesco ad impostare ad esempio se IPhone connesso a casa5g (nome della mia rete) allora in casa.

**Reply by blakdog1978** [reply]
Quante reti hai in casa ? Homeassistant e di conseguenza nmap a che rete sono connessi?

**Reply by djfoxmiki** [reply]
Ciao.
Volevo ringraziarti per l’aiuto che mi hai dato. Sono riuscito finalmente a configurare nmap e funziona divinamente.

ORa mi resta un ultima e definitiva info.

Dovrei creare un automazione su HA dove quando e solo i due smartphone (quelli configurati con nmap) non sono connessi alla rete allora far scattare un automazione che dico io.
Come imposto il tutto?

**Reply by blakdog1978** [reply]
Puoi mettere come attivazione lo stato dei dispositivi che cambi da in casa e non specifichi a ... e come condizione sempre stato che tutte e due siano fuori casa così puoi anche impostare che siano fuori casa da un lasso di tempo specifico, la fai in modalità riavvia. Prova così...

**Reply by djfoxmiki** [reply]
Che significa in modalità riavvia?

**Reply by blakdog1978** [reply]
È la modalità in cui viene eseguita l'automazione... Dai un'occhiata alle info al riguardo, riavvia significa che ogni volta che avviene l'attivazione l'automazione parte da capo... 
![IMG_20221031_194520](https://user-images.githubusercontent.com/36457031/199086225-97e4402a-9778-4c94-9737-a87a92be30d4.jpg)
![IMG_20221031_194553](https://user-images.githubusercontent.com/36457031/199086248-7cf335ae-01a0-4f3c-9667-5fd064449c6f.jpg)

**Reply by djfoxmiki** [reply]
Ciao, è normale che i dispositivi si disconnettono anche se sono regolarmente in casa? Per poi riconnettersi dopo qualche minuto?

**Reply by blakdog1978** [reply]
Non credo... Ma credo sia il tuo modem-router che crei questa cosa controlla se si disconnettono anche lì... Poi puoi aggiustare nelle impostazioni di nmap quanto tempo deve trascorrere prima di contrassegnate il dispositivo fuori casa
